pmm/uninstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+# Delete the PMM
+helm uninstall pmm
```

Comment:
this will leave PVC. Don't know how to deal with it - if this uninstall should also delete PVC or it should leave it behind for the user, but we need some message to the user that there should be additional cleanup.